src/libstd/sync/mpsc/mod.rs

```diff
@@ -1267,6 +1267,38 @@ impl error::Error for TryRecvError {
     }
 }
 
+#[stable(feature = "mpsc_recv_timeout_error", since = "1.13.0")]
+impl fmt::Display for RecvTimeoutError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
Yeah, I'm not sure either. That does seem reasonable; I'm happy to change all three to this format if that's what needs to be done.